src/server/json_family.cc

```diff
@@ -14,12 +14,17 @@
 #include <jsoncons_ext/jsonpath/jsonpath.hpp>
 #include <jsoncons_ext/jsonpointer/jsonpointer.hpp>
 #include <jsoncons_ext/mergepatch/mergepatch.hpp>
+#include <memory>
+#include <memory_resource>
```

Comment:
please do not include `<memory_resource>` directly and in general, can you please disable include what your use rule in your environment? it's too agressive